site-src/api-types/httproute.md

```diff
@@ -165,6 +165,27 @@ Service:
 Reference the [backendRef][backendRef] API documentation for additional details
 on `weight` and other fields.
 
+#### Timeouts (optional)
+
+Timeouts define API objects for L7 timeouts in HTTPRoutes. If unspecified, timeout behavior is implementation-specific.
```

Comment:
This is the _only_ reference to "L7" throughout this document. For future readers, and maybe particularly for non-native english speakers this could be slightly confusing, would this maybe be better?

```suggestion
Timeouts resources are API objects that provide request timeouts for use in `HTTPRoutes`. If unspecified, timeout behavior is implementation-specific.
```